src/styles/partials/components/_summary-list.scss

```diff
@@ -0,0 +1,30 @@
+.dg_summary-list {
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: $padding-largest;
+  text-decoration: none;
+  p {
+    color: $gray;
+    font-family: $default-heading-font;
+    font-size: $smallest-heading-font-size;
+    line-height: 30px;
+  }
+  a {
+    text-decoration: none;
+    background-color: transparent;
+  }
+  .bg-light {
```

Comment:
This can be removed, this will be handled in `.dg_summary-list`

Reply:
done